index.html

```diff
@@ -1,338 +1,257 @@
+        <a target = none href = "https://www.eff.org/cyberspace-independence">https://www.eff.org/cyberspace-independence</a> <br />
+        <a target = none href = "https://www.cnn.com/videos/tv/2020/05/30/what-is-section-230-and-how-did-it-shape-the-internet.cnn">https://www.cnn.com/videos/tv/2020/05/30/what-is-section-230-and-how-did-it-shape-the-internet.cnn</a> <br />
+        <a target = none href = "https://finbold.com/lympo-collaborates-with-binance-nft-marketplace-to-host-nfts-of-famous-athletes/">https://finbold.com/lympo-collaborates-with-binance-nft-marketplace-to-host-nfts-of-famous-athletes/</a> <br />
+        <a target = none href = "https://www.investopedia.com/non-fungible-tokens-nft-5115211">https://www.investopedia.com/non-fungible-tokens-nft-5115211</a> <br />
+        <a target = none href = "https://www.youtube.com/watch?v=Oz9zw7-_vhM">https://www.youtube.com/watch?v=Oz9zw7-_vhM</a> <br />
+        <a target = none href = "https://www.alpha-sense.com/blog/faang-q3-earnings/">https://www.alpha-sense.com/blog/faang-q3-earnings/</a> <br />
+        <a target = none href = "https://www.newsweek.com/fact-check-elon-musk-buying-twitter-led-white-house-target-section-230-1701449">https://www.newsweek.com/fact-check-elon-musk-buying-twitter-led-white-house-target-section-230-1701449</a> <br />
+        <a target = none href = "https://www.creativereview.co.uk/why-nfts-are-the-future-of-creative-expression/">https://www.creativereview.co.uk/why-nfts-are-the-future-of-creative-expression/</a><br/>
+        <a target = none href = "https://www.wsj.com/articles/how-elon-musk-can-liberate-twitter-censorship-fact-check-free-speech-buy-website-platform-takeover-content-moderation-hate-speech-misinformation-disinformation-11650983718">https://www.wsj.com/articles/how-elon-musk-can-liberate-twitter-censorship-fact-check-free-speech-buy-website-platform-takeover-content-moderation-hate-speech-misinformation-disinformation-11650983718</a><br/>
+        <a target = none href = "https://www.brookings.edu/blog/up-front/2020/07/06/how-misinformation-spreads-on-twitter/">https://www.brookings.edu/blog/up-front/2020/07/06/how-misinformation-spreads-on-twitter/</a><br/>
+        <a target = none href = "https://www.youtube.com/watch?v=jEK7P4-rDk4">https://www.youtube.com/watch?v=jEK7P4-rDk4</a><br/> 
+        <a target = none href = "https://www.americanbar.org/groups/crsj/publications/human_rights_magazine_home/the-ongoing-challenge-to-define-free-speech/in-the-age-of-socia-media-first-amendment/">https://www.americanbar.org/groups/crsj/publications/human_rights_magazine_home/the-ongoing-challenge-to-define-free-speech/in-the-age-of-socia-media-first-amendment/</a><br />
+    </ul>
+</article>
+</body>
+
+<style>
+body {
+    background-color: rgb(168, 235, 255);
+    font-weight: bold;
+    color: rgb(255, 0, 0);
+}
+

```

Comment:
ur CSS should  be declared at the top of the document, u should move the `<style> ... </style>` tags && all their contents  above the `<body>`